lib/madmin/fields/has_many.rb

```diff
@@ -17,8 +17,13 @@ def to_param
         {"#{attribute_name.to_s.singularize}_ids": []}
       end
 
-      def index_path
-        Madmin.resource_by_name(model.reflect_on_association(attribute_name).klass).index_path(format: :json)
+      def index_path(format: :json)
+        associated_resource.index_path(format: format)
+      rescue NameError
```

Comment:
This is purely because I don't have experience with the library, but I don't know why two of the `NameError`s are silent and one re-raises. Can you elaborate? It might be worth a couple of comments as well.